regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,6 +26,7 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
+        $('table').stickyTableHeaders();
```

Comment:
So the init above gets called in a $(document).ready() block when the app is loaded / first page rendered.  There aren't any ajax / partial calls on the initial load.  So the code above basically will target any table elements present in the DOM when the first page is loaded in the browser (or they hit refresh in the browser).